src/MockBot.Api/Controllers/ChatController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+
+namespace MockBot.Api.Controllers;
+
+[Route("api/chats")]
+public class ChatController : Controller
```

Comment:
```suggestion
public class ChatController : ControllerBase 
```
The Controller class inherits the ControllerBase class and adds some extra members/functions to return views which looks like are not needed in this API controller